Doc/c-api/sys.rst

```diff
@@ -289,6 +289,41 @@ accessible to C code.  They all work with the current interpreter thread's
    .. versionadded:: 3.2
 
 
+.. c:function:: int PySys_Audit(const char *event, PyObject *arguments)
```

Comment:
It would help to have an explanation for each of the parameters sort of like you did for `sys.audit()`.

Reply:
Actually, this doc is just totally out of date...